<?xml version="1.0"?>
<robot xmlns:xacro="http://ros.org/wiki/xacro" name="cu_panda">
    <xacro:include filename="$(find falcon_ros)/urdf/floating_grip.urdf.xacro"/>
    <xacro:falcon/>
  <xacro:include filename="$(find cu_panda_description)/robots/basic_setup.urdf.xacro"/>
  <joint name="falcon_to_table" type="fixed">
    <parent link="table_top"/>
    <child link="falcon_base"/>
    <origin xyz="0 0.8 0.13" rpy="${tau/4} 0 ${tau/4}"/>
  </joint>
  <link name='falcon_volume'>
    <visual>
        <geometry>
            <box size="0.1 0.1 0.1"/>        
        </geometry>
        <material name="blue_vol">
            <color rgba="0.0 0.0 1.0 0.1"/>    
        </material>
    </visual>
  </link>
  <joint name="falcon_box_to_base" type="fixed">
    <parent link="falcon_front"/>
    <child link="falcon_volume"/>
    <origin xyz="0.12 0.0 0.0" rpy="${tau/2} ${-tau/4} ${tau/4}"/>
  </joint>
</robot>